tests/specs/mcd/word-pack.k

```diff
@@ -108,4 +155,112 @@ module WORD-PACK-COMMON
 
     rule (UINT48_2 *Int pow208) |Int (maskWordPackAddrUInt48UInt48_3 &Int ADDR_UINT48_UINT48) => #WordPackAddrUInt48UInt48(maxUInt160 &Int ADDR_UINT48_UINT48, maxUInt48 &Int (ADDR_UINT48_UINT48 /Int pow160), UINT48_2) requires #rangeUInt(256, ADDR_UINT48_UINT48) andBool #rangeUInt(48, UINT48_2) [simplification]
     rule (UINT48_1 *Int pow160) |Int (maskWordPackAddrUInt48UInt48_2 &Int ADDR_UINT48_UINT48) => #WordPackAddrUInt48UInt48(maxUInt160 &Int ADDR_UINT48_UINT48, UINT48_1, maxUInt48 &Int (ADDR_UINT48_UINT48 /Int pow208)) requires #rangeUInt(256, ADDR_UINT48_UINT48) andBool #rangeUInt(48, UINT48_1) [simplification]
+
+    rule maxUInt48 &Int (maskWordPackUInt48UInt48_2 &Int A) => 0 requires 0 <=Int A [simplification]
+
+    //
+    // #WordPack decomposition
+    // 
+    rule       ADDR_UINT48_UINT48:Int ==Int #WordPackAddrUInt48UInt48(ADDR:Int, UINT48_1:Int, UINT48_2:Int) 
+            => ADDR     ==Int maxUInt160 &Int  ADDR_UINT48_UINT48              
+       andBool UINT48_1 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow160)
+       andBool UINT48_2 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow208)
+       andBool (0 <=Int ADDR_UINT48_UINT48) andBool (ADDR_UINT48_UINT48 <Int pow256)
+       andBool (0 <=Int ADDR)               andBool (ADDR     <Int pow160) 
+       andBool (0 <=Int UINT48_1)           andBool (UINT48_1 <Int pow48)
+       andBool (0 <=Int UINT48_2)           andBool (UINT48_2 <Int pow48)
+       [simplification]
```

Comment:
Is this still needed given the lemmas above which handle `{... #Equals #WordPack... }`? It seems that those ones should superscede these ones, especially given: https://github.com/runtimeverification/k/pull/2659

Reply:
It's not, the tests are passing without them. I'll move them to the Java-related module.